apigee/apigee-api-catalog-drupal
# #80 Upgrade path from API doc custom entities to nodes

This provides an upgrade path from the 1.x version of the module that uses custom entities to the 2.x version that uses nodes (see #84 ). 

- Install the node type and fields
- Recreate any custom fields that were added to the API Doc entity onto the new node bundle.
- Migrate data
- Delete all API doc entities and remove all of the outdated definitions.

To test:
Using the module in the 1.x version with some API Doc entities created, switch to this branch and run database updates. It should migrate all entities into nodes.
